apis/addonconfigs/cni/v1alpha2/antreaconfig_types.go

```diff
@@ -288,51 +288,169 @@ type AntreaNsx struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:=false
 	Enable bool `json:"enable,omitempty"`
-	// BootstrapFrom either providerRef or inline configs
-	// +kubebuilder:validation:Optional
-	BootstrapFrom AntreaNsxBootstrapFrom `json:"bootstrapFrom,omitempty"`
```

Comment:
We changed the CRD, so need to generate manifest and commit the manifest changes.